### ISSUE TITLE
Configure final directory for each temporary folder

### ISSUE THREAD

**Issue by koninka** [ericaltendorf/plotman#578]
I connected two ssd drives to my plotter. And I would like not to transfer the plots to the final directory `-d` ([skip phase 4](https://github.com/Chia-Network/chia-blockchain/wiki/FAQ#what-is-the-secondary-temp-directory--2-and-how-should-i-set-it)), but immediately `archive` (rsync) them from ssd to remote driver.

But I cannot do this, since the plotman can select the final directory on another ssd and I will have unnecessary transfer operations from one ssd to another.

So It turns out that the scenario does not work , when I use a plotter machine and plot on independent ssd without a final directory and immediately transfer it with rsync.

**Comment by altendky** [comment]
Based on your config over in https://github.com/ericaltendorf/plotman/issues/487#issuecomment-842811252, I think you want the new feature (only on development at the moment, not main/release) where you can comment out `dst` section entirely and the plot will stay on the ssd it was created on and archive will pick it up from there.  This is how I am presently running.

**Comment by syadnom** [comment]
I'm just mixing mine in here because we're all having similar thoughts
https://github.com/ericaltendorf/plotman/issues/616

creating tmp/dst pairs would be fine for me, or just completely omitting dst to lock tmp and dst to the same drive.  

If this is already in dev as altendky says then It looks like I'm probably all set when that arrives.

**Comment by koninka** [comment]
@syadnom agree with you. Right now (`development` branch) we can ommit dst directory and plot will be saved on the same drive.

But if I want to manage from which ssd where to transfer, then this is the problem.

But nevertheless, I still am glad that now I can omit the final directory and there will be no unnecessary copying operations

**Comment by syadnom** [comment]
I think that will get improved substantially with archiving a bit as they are reworking that.  Still doesn't directly solve your want to map tmp to final destination explicitly...

**Comment by syadnom** [comment]
koninka, how solid is the dev branch?  I'd really like to use this feature.

Also, if you do this as-is right now, does plotman still see the plots for archiving?

I'm wanting to tmp>dst on same drive and then fire the archiver.  If that's not workable I'll stick with my external rsync script.

**Comment by koninka** [comment]
@syadnom I've been working on the `dev` branch for a week now and everything works well.

And archiving operation works correct.

**Comment by syadnom** [comment]
this is the @development branch right?

**Comment by syadnom** [comment]
yep, it is.  I'm testing now.  Looks good so far.  I like that dst is just a copy of the tmp.    I'm going to run this on my plot and archive boxes.  I expect some contention during the archive because it'll spin up another plot on the same disk, but no worse than the redundant copy.

I'd do it on the plot 'n store boxes but the archiver is single threaded so I'll keep using my aufs/mfs setup there.
